content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,314 @@
 ---
 layout: "enterprise"
-page_title: "Logging - Infrastructure Administration - Terraform Enterprise"
+page_title: "Log Forwarding - Infrastructure Administration - Terraform Enterprise"
 ---
 
-# Terraform Enterprise Logs
+# Terraform Enterprise Log Forwarding
 
-This document contains information about interacting with Terraform Enterprise logs. There are two types of logs, application logs and audit logs. Application logs emit information about the services that comprise Terraform Enterprise. Audit logs emit information whenever any resource managed by Terraform Enterprise is changed.
+Terraform Enterprise supports forwarding its logs to one or more external
+destinations, aptly referred to as log forwarding. This page details how to
+configure log forwarding in Terraform Enterprise.
 
-## Application Logs
+## Requirements
 
-Terraform Enterprise runs in a set of Docker containers. As such, any tooling that can interact with Docker logs can read the logs. This includes the command `docker logs`, as well as access via the [Docker API](https://docs.docker.com/engine/api/v1.36/#operation/ContainerLogs).
+Log forwarding requires:
 
-An example of a tool that can automatically pull logs for all docker containers is [logspout](https://github.com/gliderlabs/logspout).
-Logspout can be configured to take the Docker logs and send them to a syslog endpoint. Here's an example invocation:
+- Terraform Enterprise running on an instance with the `systemd-journald`
```

Comment:
If it's not enabled, should we tell them how to do that, or will they already know?

Reply:
I'm going to chew on this a bit. I don't want to overload that list item. Technically if a user is running on a support operating system then they should already be using `systemd-journald`. Leaving this open while I work on it.

Reply:
gotcha! Just let me know when it's ready for me to look again :-)

Reply:
I changed up the wording a tiny bit and decide to leave out instructions on starting and enabling the `systemd-journald` service. That service should be started and enabled by default on our supported operating systems and if it is not then that's an outlier that should probably be funneled through the proper Customer Success channels.

Reply:
gotcha okay! works for me!